internal/http_api/api_request.go

```diff
@@ -121,7 +120,7 @@ retry:
 
 func httpsEndpoint(endpoint string, body []byte) (string, error) {
 	var forbiddenResp struct {
-		HTTPSPort int `json:"https_port"`
+		HTTPSAddr string `json:"https_addr"`
```

Comment:
this is the other half, of the part that I think should be reverted, for backwards compatibility

Reply:
Thank you, done